pyplanet/apps/contrib/voting/__init__.py

```diff
@@ -183,6 +197,7 @@ async def map_start(self, *args, **kwargs):
 		"""
 
 		self.podium_stage = False
+		self.extend_current_count = 0
```

Comment:
Is there a need to reset the variable both in ``podium_start`` and ``map_start``?

Reply:
i don't see how it works without, you're free to remove it.